### ISSUE TITLE
线性模型收敛速度太慢

### ISSUE THREAD

**Issue by whtwhtw** [PaddlePaddle/Paddle#48781]
### bug描述 Describe the Bug

构造一个y=5x+3+rand()/10的单层，单变量，单输出的回归模型，数据量100对。迭代20000个epochs，loss还是很大（200以上），模型依然没有收敛。对比tensorflow的loss已经在8.6e-04级别了，权重已经很准确。
具体代码如下：
# 定义数据集
class line_Dataset(Dataset):
    def __init__(self, num_samples):
        self.num_samples = num_samples

    def __getitem__(self, idx):
        x = np.arange(0, 10, 0.1).reshape(-1, 1).astype('float32')
        y = (5*x + 3 + np.random.random(100).reshape(-1, 1)).T.flatten().astype('float32')
        return x, y

    def __len__(self):
        return self.num_samples

dataset = line_Dataset(1)

# 定义模型
class LineRegulation(pd2.nn.Layer):
    def __init__(self):
        super(LineRegulation, self).__init__()
        self.fc = pd2.nn.Linear(1, 1)

    def forward(self, input_data):
        pred = self.fc(input_data)
        return pred

# 创建模型
model = pd2.Model(LineRegulation())

# 模型训练
optim = pd2.optimizer.Adam(learning_rate=0.01, parameters=model.parameters())
model.prepare(optim, pd2.nn.MSELoss())
model.fit(dataset, epochs=20000, batch_size=1000, verbose=1)

model.summary((1,))
print(model.parameters())

### 其他补充信息 Additional Supplementary Information
tensorflow代码对比如下：
model = tf.keras.Sequential([tf.keras.layers.Dense(1)])
model.build(input_shape=(1, 1))
model.compile(optimizer='adam', loss='mse')
history = model.fit(x, y, epochs=20000, batch_size=1000, verbose=1)

**Comment by whtwhtw** [comment]
细微处见真张，真的希望paddle越来越强大

**Comment by Shixiaowei02** [comment]
您好，问题已收到，我们正在尝试复现。谢谢您的支持
